ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSemanticAnalyzer.java

```diff
@@ -234,6 +235,56 @@ private void initReplLoad(ASTNode ast) throws SemanticException {
     }
   }
 
+  private boolean filterEvent(FileStatus dir, DumpType dumpType) throws SemanticException{
```

Comment:
can rename this method as isEventApplied().

Reply:
changed the name to shouldReplayEvent as for some event we replay it even if its applied